### ISSUE TITLE
Excuse me, a small request

### ISSUE THREAD

**Issue by Gooooe** [mikestock/intf-tools#1]
I recently spent a lot of time learning your code, during this time I learned a lot of new knowledge, I also learned your paper, but I still don't understand some algorithms in the code. I think it is best to learn from a simple data.
Can you upload a copy of the original data, good or bad, I would like to use this data to better understand your algorithm.

**Comment by mikestock** [comment]
The data for this stuff is pretty big, around 1.5GB/second, far too large to add to the git repository.  I'll see if I can maybe get a shorter sample included.  May be a bit though.

I've been thinking about how to go about making a lot of this available anyway.  It's been 7 years since I recorded it, seems fair that if someone wants to try doing something new and interesting with the data they should be able to.

**Comment by Gooooe** [comment]
That's great!

If I can put these codes into operation, it will undoubtedly be a great sense of achievement for me.

I can see that the code was uploaded 9 years ago, it may be a little difficult for you to find the raw data, but if you would like to help me, I would appreciate it!

Including your recently uploaded LMA positioning algorithm, these are very valuable learning resources for me.

**Comment by mikestock** [comment]
The LMA location algorithm is far from complete, I only have portions of a raw data-reader done so far.  I suspect it will be a few months at least before I have it working, and longer before I have it working well.  But, the imaging code for the LWA (another radio interferometer) is functional if slow

**Comment by Gooooe** [comment]
Yes, I have looked at the LMA code. I'm looking forward to how it will look when finished. I think this is a great contribution for many Lightning enthusiasts.

I'm still continuing to learn INTF code today, and I think I'm learning a really cool part of this project, after starting the thread, running the run function, and the event class called in the merge thread later. This part of the code is a bit difficult for me to understand and may need to be understood in conjunction with your paper. I think it would be easier to understand with an example that runs the code.

I would love to learn these techniques, which excites me.

**Comment by mikestock** [comment]
The portion of the code you're looking at now is what enables processing on multiple CPUs, python's support for this isn't great.  Because the processing happens asynchronously, it can be a little bit hard to follow.  the lwali project does very similar VHF imaging of lightning but is single threaded if you want to see a simpler example.
